aggregator/src/app/BundleRouter.ts

```diff
@@ -24,6 +24,16 @@ export default function BundleRouter(bundleService: BundleService) {
   router.get(
     "bundleReceipt/:hash",
     async (ctx) => {
+      // check pending
```

Comment:
Return 202 error message if the bundle is in the DB.  Also, pass a submit error in the body